pandas/io/parsers.py

```diff
@@ -3782,3 +3684,59 @@ def _make_reader(self, f):
             self.skiprows,
             self.infer_nrows,
         )
+
+
+def _check_defaults_read(read_kwds: dict, defaults: dict):
```

Comment:
can we instead pass by keyword args (you can still pass the defaults as a dict i think), type is using Dict[str, Any] though

Reply:
Ok